### PR TITLE
Changes to elimiate session status and normalization changes on asses…

### PR DESCRIPTION
**Pull request by sumathi-thirumani**
…sment_student

### CODE REVIEW

File: api/src/main/java/ca/bc/gov/educ/eas/api/controller/v1/AssessmentStudentController.java [hunk 1]
```diff
@@ -27,13 +27,14 @@ public class AssessmentStudentController implements AssessmentStudentEndpoint {
   private final AssessmentStudentService studentService;
   private final AssessmentStudentValidator validator;
   private final AssessmentStudentSearchService searchService;
+  private final AssessmentStudentMapper mapper;
```
Comment (arcshiftsolutions) [inline api/src/main/java/ca/bc/gov/educ/eas/api/controller/v1/AssessmentStudentController.java]: No need to inject the mapper - just use AssessmentStudentMapper.mapper.

File: api/src/main/java/ca/bc/gov/educ/eas/api/repository/v1/AssessmentRepository.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@Repository
+public interface AssessmentRepository extends JpaRepository<AssessmentEntity, UUID> {
+
+    @Query(value = "SELECT ASST FROM AssessmentEntity ASST WHERE ASST.sessionEntity.sessionID=:sessionID AND ASST.assessmentTypeCode =:assessmentTypeCode")
```
Comment (arcshiftsolutions) [inline api/src/main/java/ca/bc/gov/educ/eas/api/repository/v1/AssessmentRepository.java]: Remove

**Review by arcshiftsolutions — APPROVED** [review]

File: api/src/main/java/ca/bc/gov/educ/eas/api/controller/v1/AssessmentStudentController.java (hunk 1 above)
Comment (sumathi-thirumani) [reply api/src/main/java/ca/bc/gov/educ/eas/api/controller/v1/AssessmentStudentController.java]: Changes made.

File: api/src/main/java/ca/bc/gov/educ/eas/api/repository/v1/AssessmentRepository.java (hunk 2 above)
Comment (sumathi-thirumani) [reply api/src/main/java/ca/bc/gov/educ/eas/api/repository/v1/AssessmentRepository.java]: Query removed.

**Pull request merged**